### ISSUE TITLE
react-hamburger hides the heading in the generate profile route in the pc version.

### ISSUE THREAD

**Issue by saswat10** [khairalanam/GPRG#14]
![Screenshot 2023-10-04 195847](https://github.com/khairalanam/GPRG/assets/104200293/8da195a0-fd4c-433d-b8ef-4e67c6e4fb94)

**Comment by sasam10** [comment]
is this issue happening only with me or someone else is also facing the issue.

**Comment by nayakswadhin** [comment]
> is this issue happening only with me or someone else is also facing the issue.
I think You should provide proper padding top in the page.

**Comment by khairalanam** [comment]
I believe the issue is that the Nav element has been put in fixed mode. It shouldn't be fixed @nayakswadhin

**Comment by nayakswadhin** [comment]
> I believe the issue is that the Nav element has been put in fixed mode. It shouldn't be fixed @nayakswadhin
@khairalanam Yup its true but if we use relative or some other positional property then the content is of the page is also going down/up depending on the navbar. So if it's ok then we can change otherwise one more solution is we can give add padding top on every page or add in global CSS.

https://github.com/khairalanam/GPRG/assets/114871538/5f14ffd6-e142-4639-a4c1-fba146e02d17

**Comment by khairalanam** [comment]
The navbar doesn't have to be sticky or fixed since there's not much content in any of the pages except the form page but I doubt many of them would scroll up and then go to some nav link in the midst of filling a form.

If the fixed property is necessary, try putting it in only in mobile and/or tablet view but when it comes to lg breakpoint, you can change from fixed to normal or relative mode.

**Comment by nayakswadhin** [comment]
@khairalanam If we put navbar fixed only for mobile/tablet view then that view will have the same issue. So i guess our only solution here will be to convert the fixed property to normal/relative for both mobile and laptop view, right??

**Comment by khairalanam** [comment]
That's also fine. Either way, navbar in its sticky or fixed mode is not at all necessary.

**Comment by nayakswadhin** [comment]
> That's also fine. Either way, navbar in its sticky or fixed mode is not at all necessary.

@khairalanam I have raised a pr on this issue. Kindly review it.
